test/babelish/test_csv2android.rb

```diff
@@ -24,10 +24,10 @@ def test_converting_csv_with_special_chars
 
   def test_converting_csv_to_dotstrings_one_output_option
     csv_file = "test/data/test_data.csv"
-    single_file = 'myApp.xml'
+    single_file = "myApp.xml"
     converter = Babelish::CSV2Android.new(csv_file,
-    {'English' => "en"},
-    :output_basename => 'myApp',
+    {"English" => "en"},
```

Comment:
@borut-t I think it means it should be
```
{ "English" => "en" }
```
the rules are based with Rubocop's guideline : https://github.com/bbatsov/ruby-style-guide